src/data-table/DataTableBody.tsx

```diff
@@ -78,6 +78,7 @@ export function DataTableBody<T extends ReferenceObject>(props: DataTableBodyPro
         const primaryAction = _(availableActions).find({ primary: true }) || defaultAction;
 
         const contextualAction = (event: MouseEvent<unknown>) => {
+            if (isEventCtrlClick(event)) return;
```

Comment:
This fixes the control-click issue we have when working with Cypress FD recorder